### DISCUSSION TITLE
Sub-collections or combining collections

### DISCUSSION THREAD

**Discussion by Blastinburn** [bcssov/IronyModManager#225] [Ideas]
I think it would be useful to be able to combine collections or include collections in each other when loading the game.  The idea being you could have a base set of mods you always run (Ex. quality of life and UI mods) and are combined into mods you are using for specific games/playthroughs.

I am currently in several different multiplayer stellaris games that each use different modlists as well as personal games and I want all of them to use the same local list of UI mods. (Ex. UI overhaul, tiny UI)  The problem is when I add another local mod I often forget to go through every collection to add it.  This is especially a problem with tiny UI since there are a lot of variants and sub-mods that do slightly different things and may or may not work depending on load order. (I've gotten tiny fleets to work exactly once and never again.  I also keep forgetting to replace tiny ship designer with a different version.)

The idea would be that I could maintain some kind of "always" collection and have that included in another collection with additional mods, so I only need to make changes to a single list for UI mods. 

Thank you for your time, Irony has been a great help in managing mods and keeping the game stable and quick.

**Comment by bcssov** [comment]
I like the idea but it's actually a lot of work. Lots of UI changes, lots of backend changes to make this work and the need for a better UX. Going to leave this here hopefully for some future update.
